storage/statedb/cache_fastcache.go

```diff
@@ -52,8 +52,9 @@ func NewFastCache(cacheSizeMB int) TrieNodeCache {
 		return nil
 	}
 
-	logger.Info("Initialize local trie node cache (fastCache)", "MaxMB", cacheSizeMB)
-	return &FastCache{cache: fastcache.New(cacheSizeMB * 1024 * 1024)} // Convert MB to Byte
+	logger.Info("Initialize local trie node cache (fastCache)",
+		"filePath", filePath, "MaxMB", cacheSizeMB)
+	return &FastCache{cache: fastcache.LoadFromFileOrNew(filePath, cacheSizeMB*1024*1024)} // Convert MB to Byte
```

Comment:
```suggestion
	return &FastCache{cache: fastcache.LoadFromFileOrNew(filePath, cacheSizeMB * 1024 * 1024)} // Convert MB to Byte
```

Reply:
@yoomee1313 It seems that linter removes unnecessary whitespaces 😅